gapis/api/gles/compat.go

```diff
@@ -562,6 +562,13 @@ func compat(ctx context.Context, device *device.Instance) (transform.Transformer
 				log.E(ctx, "Error decompressing texture: %v", err)
 			}
 
+		case *GlTexBufferEXT:
+			if !version.IsES && version.AtLeast(3, 1) { // Strip suffix on desktop.
```

Comment:
The reason I ask to have it, is that it's easier to see a single untransformed command that errors with "Attempted to unsupported function XXX", than try and trace through compat to figure out what things could have produced the call in the first place.
By annotating these transformations with the target GL version, it's clear what we're expecting from a replay device. At the moment, I can't just look at the code and figure out what the maximum GL version we need for all compat functions to work. Longer term, I'd hope that we'd correctly handle the GLES and GL paths, and also know ahead of time when said device won't replay - that's what [`GetReplayPriority`](https://github.com/google/gapid/blob/73a534ddb2aff14c1302f55bb296599ef154cb0e/gapis/replay/interfaces.go#L28-L34) should really be returning.